code/modules/martial_arts/cqc.dm

```diff
@@ -141,8 +140,9 @@
 	add_to_streak("H", D)
 	if(check_streak(A, D))
 		return TRUE
+	add_attack_logs(A, D, "Melee attacked with martial-art [src]", ATKLOG_ALL)
 	A.do_attack_animation(D)
-	var/picked_hit_type = pick("CQC'd", "neck chopped", "gut punched")
+	var/picked_hit_type = pick("CQC'd", "Big Bossed")
```

Comment:
the sound for CQC is ripped straight from the game. Sorry, yer gunna get Big Bossed.

Reply:
i mean i thought i reviewed it right..but..eh..maybe we need new sounds thing though..kina don't agree here. but thats me

Reply:
We specifically decided on this, no.

Reply:
This is just about the only part of the PR I'd like to see in the game